tests/Console/ConfigurationResolverTest.php

```diff
@@ -954,6 +954,123 @@ public function testResolveRulesWithOption(): void
+            ],
+            [
+                'The rules contain unknown fixers: "no_multiline_whitespace_before_semicolons" is a renamed rule,  did you mean "multiline_whitespace_before_semicolons"?
+
+For more info see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#renamed-rules.',
+                'no_multiline_whitespace_before_semicolons',
+            ],
+            [
+                'The rules contain unknown fixers: "no_short_echo_tag" is a renamed rule,  did you mean "echo_tag_syntax"? note: use configuration [\'format\' => \'long\']
+
+For more info see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#renamed-rules.',
+                'no_short_echo_tag',
+            ],
+            [
+                'The rules contain unknown fixers: "php_unit_ordered_covers" is a renamed rule,  did you mean "phpdoc_order_by_value"? note: use configuration [\'annotations\' => [\'covers\']]
+
+For more info see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#renamed-rules.',
+                'php_unit_ordered_covers',
+            ],
+            [
+                'The rules contain unknown fixers: "phpdoc_inline_tag" is a renamed rule,  did you mean "general_phpdoc_tag_rename, phpdoc_inline_tag_normalizer and phpdoc_tag_type"?
+
+For more info see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#renamed-rules.',
+                'phpdoc_inline_tag',
+            ],
+            [
+                'The rules contain unknown fixers: "pre_increment" is a renamed rule,  did you mean "increment_style"? note: use configuration [\'style\' => \'pre\']
+
+For more info see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#renamed-rules.',
+                'pre_increment',
+            ],
+            [
+                'The rules contain unknown fixers: "psr0" is a renamed rule,  did you mean "psr_autoloading"? note: use configuration [\'dir\' => \'x\']
+
+For more info see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#renamed-rules.',
+                'psr0',
+            ],
+            [
+                'The rules contain unknown fixers: "psr4" is a renamed rule,  did you mean "psr_autoloading"?
+
+For more info see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#renamed-rules.',
+                'psr4',
+            ],
+            [
+                'The rules contain unknown fixers: "silenced_deprecation_error" is a renamed rule,  did you mean "error_suppression"?
+
+For more info see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#renamed-rules.',
+                'silenced_deprecation_error',
+            ],
+            
+        ];
+    }
+
+    /**
+     * Test the renamed rule trailing_comma_in_multiline_array
+     * 
+     * @return void
+     */
+    public function testResolveRenamedRulesTrailingCommaInMultilineArrayWithUnknownRules(): void
```

Comment:
why not part of `provideRenamedRulesCases` ?

Reply:
I forgot to add that one. Fixed in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5887/commits/5944eedd319ef4d2cd7ffa17e29139a52cfa7941